src/sfizz/Synth.cpp

```diff
@@ -676,6 +676,9 @@ void sfz::Synth::setSampleRate(float sampleRate) noexcept
 void sfz::Synth::renderVoiceToOutputs(Voice& voice, AudioSpan<float>& tempSpan) noexcept
 {
     const Region* region = voice.getRegion();
+    if (region == nullptr)
```

Comment:
How does it happen that a running voice is not associated with a region?

Reply:
Could be an assert, I had trouble when working on this at some point where voice that were already "killed" were trying to be rendered, it's not necessary anymore.